package.json

```diff
@@ -12,16 +12,16 @@
     "watch": "mocha --recursive --watch test",
     "docs": "jsdoc2md index.js"
   },
-  "homepage": "https://github.com/noblesamurai/express-amqp-logger",
+  "homepage": "https://github.com/noblesamurai/node-log2amqp",
   "repository": {
     "type": "git",
-    "url": "git://github.com/noblesamurai/express-amqp-logger.git"
+    "url": "git://github.com/noblesamurai/node-log2amqp.git"
   },
   "bugs": {
-    "url": "https://github.com/noblesamurai/express-amqp-logger/issues"
+    "url": "https://github.com/noblesamurai/node-log2amqp/issues"
   },
   "engines": {
-    "node": "6.x",
+    "node": ">=0.4.0 <8",
```

Comment:
Actually, I think you mean `>=4` as you use es2015 features like `let` which won't work in node `0.4.0`.